cis_enforcements/azure/cis_v200/section8/policies.tf

```diff
@@ -0,0 +1,96 @@
+# Azure > KeyVault > Vault > Approved
+resource "turbot_policy_setting" "azure_keyvault_vault_approved" {
+  resource = turbot_smart_folder.main.id
+  type     = "tmod:@turbot/azure-keyvault#/policy/types/vaultApproved"
+  note     = "Azure CIS v2.0.0 - Control: 8.6 and 8.7"
+  value    = "Check: Approved"
+  # value    = "Enforce: Delete unapproved"
+}
+
+# Azure > KeyVault > Vault > custom
+resource "turbot_policy_setting" "azure_keyvault_vault_approved_custom" {
+  resource       = turbot_smart_folder.main.id
+  type           = "tmod:@turbot/azure-keyvault#/policy/types/vaultApprovedCustom"
+  note           = "Azure CIS v2.0.0 - Control: 8.6 and 8.7"
+  template_input = <<-EOT
+    {
+      vault {
+        enableRbacAuthorization: get(path:"properties.enableRbacAuthorization"),
+        privateEndpointConnections: get(path:"properties.privateEndpointConnections")
+
+      }
+    }
+  EOT
+  template       = <<-EOT
+    {% set results = [] -%}
+
+    {%- if $.vault.enableRbacAuthorization == false -%}
+
+      {%- set data = {
+          "title": "Role Based Access Control",
+          "result": "Not approved",
+          "message": "Role based access control is disabled"
+      } -%}
+
+    {%- else -%}
+
+      {%- set data = {
+          "title": "Role Based Access Control",
+          "result": "Approved",
+          "message": "Role based access control is enabled"
+        } -%}
```

Comment:
```suggestion
    {%- elif $.vault.enableRbacAuthorization == false -%}

      {%- set data = {
          "title": "Role Based Access Control",
          "result": "Approved",
          "message": "Role based access control is enabled"
        } -%}
        
    {%- else -%}

      {%- set data = {
          "title": "Role Based Access Control",
          "result": "Skip",
          "message": "No data for role based access control yet"
        } -%}
```

Reply:
The value of `enableRbacAuthorization` is either true or false, so `No data` condition will not occur.